docs/platform/integration/commercetools/features.md

```diff
@@ -0,0 +1,57 @@
+---
+id: features
+title: commercetools features
+sidebar_label: Features
+---
+
+## Supported Features
+
+We have covered all features essential for customer shopping and we keep improving our feature coverage over time.
+
+**General**
+
+- Top navigation menu
+- Multiple stores support (multilanguage support)
+- Multiple currency support
+
+**Catalog / Product list**
+
+- Layered navigation
+- Product lazy load
+- Sorting
+
+**Catalog / Product Detail Page**
+
+- Product images & gallery
+- Sale Prices
+- Configurable product: swatches
+
+**Customer**
+
+- Login
+- Register
+- Reset Password
+- Address Book
+- Customer Dashboard
+- Account Details
+- Orders
+
+**Cart**
+
+- Add / Remove / Update Items
+- Add / Remove Coupons
+
+**Checkout**
+
+- Guest / Customer Checkout
+- Adding new address / reusing existing ones
+- Shipping methods
+- Various payment methods
+- Gift cards support
```

Comment:
```suggestion
- Gift cards support (checkout only)
```

As far as I'm aware we only support using them, not selling them or viewing them in the customer dashboard. I know this is in the checkout section but i think it's important to specify this.